bin/main.rb

```diff
@@ -0,0 +1,82 @@
+require_relative '../lib/game.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
+
+# welcome info
+
+puts
+puts "*" * 27
+puts "  WELCOME TO TIC-TAC-TOE!"
+puts "*" * 27
+puts
+
+puts 'What is your name player 1?'
+name_1 = gets.chomp
+
+puts 'What is your name player 2?'
+name_2 = gets.chomp
+
+#initialization
+
+player1 = Player.new(name_1, 1)
+player2 = Player.new(name_2, 2)
+board = Board.new
+game = Game.new(player1.name, player2.name)
+
+
+#game
+
+puts 'Let\'s play!'
+
+count = 0
+
+while count < 9
+
+  puts board.display_board
+
+  if count.even?
+
+    puts 'Choose the number where you want to place your move'
```

Comment:
[OPTIONAL]
This should reference which player has to move, it can get confusing and was a requirement of the previous milestone (it is required that the game tells which player's turn it is! Whether you reference by the symbol or their name)

Please consider this before merging nad moving  on 🙏